src/translations/enTranslations.json

```diff
@@ -85,16 +85,11 @@
       "show_full_nin": "Show national identity number"
     },
     "users": {
-      "add_role_info": "If you don't find the user, you must check that they are affiliated with your institution and have previously logged in.",
       "area_of_responsibility": "Area of responsibility",
       "area_of_responsibility_placeholder": "No limitations",
```

Comment:
Denne er fjernet i norsk fil, men ikke her. Tips for å sjekke at språkene er i sync: se at begge filer har like linjenummer